doc/source/reference/global_state.rst

```diff
@@ -82,4 +82,4 @@ debug option can be interesting for testing code written
 in C which iterates through arrays that may or may not be
 contiguous in memory.
 Most users will have no reason to change these, for details
-please see the `memory layout <memory-layout>`_ documentation.
```

Comment:
This label exists. Perhaps this needs `:ref:`, or something has gone wrong with numpy-doc?

https://github.com/numpy/numpy/blob/3e261be81a5f2645ffc4c1e79a5dccc65a216669/doc/source/reference/arrays.ndarray.rst#L90-L93

Reply:
Thanks, will change:
```
see the :ref:`memory layout <memory-layout>` documentation.
```